pkg/services/auth/settings.go

```diff
@@ -0,0 +1,121 @@
+package auth
+
+import (
+	"io/ioutil"
+	"sync"
+
+	"github.com/BurntSushi/toml"
+	"golang.org/x/xerrors"
+
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/setting"
+	"github.com/grafana/grafana/pkg/util/errutil"
+)
+
+// AuthConfig holds list of connections to LDAP
+type AuthConfig struct {
+	AuthMappings []*AuthOrgConfig `toml:"auth"`
+}
+
+// AuthOrgConfig holds connection data to LDAP
+type AuthOrgConfig struct {
+	Groups []*GroupToOrgRole `toml:"group_mappings"`
+}
+
+// GroupToOrgRole is a struct representation of LDAP
+// config "group_mappings" setting
+type GroupToOrgRole struct {
+	GroupDN string `toml:"group_dn"`
+	OrgID   int64  `toml:"org_id"`
+
+	// This pointer specifies if setting was set (for backwards compatibility)
```

Comment:
What's `for backwards compatibility`? Do we actually need this?

Reply:
That was not my comment.

Reply:
`git blame` disagrees...

Reply:
Well, yes. In the new file for auth settings.  I am keeping the same data structure as `pkg/services/ldap/settings.go`  where that comment originates from.

Reply:
LGTM